azure-pipelines.yml

```diff
@@ -68,6 +68,38 @@ stages:
       value: $(Build.SourceBranchName)
 
   jobs:
+  - job: get_impacted_area
+    displayName: "Get impacted area"
+    timeoutInMinutes: 240
+    continueOnError: true
+#    pool: sonic-ubuntu-1c
+    steps:
+      - template: .azure-pipelines/impacted_area_testing/get-impacted-area.yml
+
+  - job: impacted_area_t0_elastictest
+    displayName: "kvmtest-t0-on-needed by Elastictest"
+    dependsOn: get_impacted_area
+    condition: or(contains(dependencies.get_impacted_area.outputs['SetVariableTask.PR_CHECKERS'], 't0'), eq(dependencies.get_impacted_area.outputs['SetVariableTask.PR_CHECKERS'], ''))
+    variables:
+      TEST_SCRIPTS: $[ dependencies.get_impacted_area.outputs['SetVariableTask.TEST_SCRIPTS'] ]
+    timeoutInMinutes: 240
+    continueOnError: true
+    pool: sonic-ubuntu-1c
+    steps:
+      - template: .azure-pipelines/impacted_area_testing/calculate-instance-numbers.yml
+        parameters:
+          TOPOLOGY: t0
+          BUILD_BRANCH: $(BUILD_BRANCH)
+
+      - template: .azure-pipelines/impacted_area_testing/impacted-area-elastictest-template.yml
+        parameters:
+          TOPOLOGY: t0
+          SCRIPTS: $(SCRIPTS)
+          MIN_WORKER: $(INSTANCE_NUMBER)
+          MAX_WORKER: $(INSTANCE_NUMBER)
```

Comment:
It's not a environment variable, it's the output from `impacted-area-elastictest-template.yml`

Reply:
yeah, if the pre-step fail, do we have an default value of `instance_number` to always run the test like before.
As well as `scripts`.

Reply:
I think if the pre-step fail, we will not continue the pipeline, just fail here. As the default instance number is not suitable, and it may cause too much time running PR testing.